packages/core/email/package.json

```diff
@@ -19,7 +19,26 @@
       "url": "https://strapi.io"
     }
   ],
+  "files": [
+    "./dist",
+    "strapi-server.js"
+  ],
+  "exports": {
+    "./strapi-admin": {
+      "types": "./dist/admin/index.d.ts",
+      "source": "./admin/src/index.ts",
+      "import": "./dist/admin/index.mjs",
+      "require": "./dist/admin/index.js",
+      "default": "./dist/admin/index.js"
+    },
```

Comment:
I will add the BE stuff in the other branch or the main feature branch once merged, adding it here caused the build to fail